lib/commands/framework-versions/print-versions.ts

```diff
@@ -26,8 +27,10 @@ export class PrintFrameworkVersionsCommand implements ICommand {
 
 	public canExecute(args: string[]): IFuture<boolean> {
 		return (() => {
-			this.$project.ensureCordovaProject();
-
+			this.$project.ensureProject();
+			if(this.$project.projectData.Framework !== this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova && this.$project.projectData.Framework !== this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.NativeScript) {
```

Comment:
Can we add new property to `projectCapabilities` ? Actually only `website` project should throw error, `cordova` and `nativescript` should return true.